client/tackle.lua

```diff
@@ -4,7 +4,7 @@ lib.addKeybind({
     defaultKey = 'E',
     onReleased = function(self)
         if cache.vehicle then return end
-        if QBX.PlayerData.metadata.ishandcuffed then return end
+ 	if QBX.PlayerData.metadata.ishandcuffed then return end
```

Comment:
Indentation is off

Reply:
Looks fine on my screen?

<img width="1130" alt="Screenshot 2023-09-28 at 7 02 57 AM" src="https://github.com/Qbox-project/qbx_smallresources/assets/3579092/773e31ec-c270-4647-9b70-762a15137958">

Reply:
![Screenshot_20230928_160447_DuckDuckGo](https://github.com/Qbox-project/qbx_smallresources/assets/82737367/9f7b886e-be85-41c1-9a9a-1dd5bd31ed6b)

Reply:
Could that be a GitHub issue? Try looking at the code on branch, it doesn't seem to match the diff.

Reply:
<img width="428" alt="Screenshot 2023-09-28 at 7 10 00 AM" src="https://github.com/Qbox-project/qbx_smallresources/assets/3579092/f7d94600-00ce-4e01-9da7-bc9b55c219c1">

Reply:
> Could that be a GitHub issue? Try looking at the code on branch, it doesn't seem to match the diff.

There is a difference, you might not see it on desktop, but the differencd here is tab and space